packages/blockchain-wallet-v4-frontend/src/scenes/Dex/hooks/useSceneResolver.ts

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react'
```

Comment:
This hook is something I would consider refactoring into Redux state + saga in the future work.

Reply:
TBD